doc/whats-new.rst

```diff
@@ -212,6 +212,11 @@ Bug fixes
   the first argument was a numpy variable (:issue:`1588`).
   By `Guido Imperiale <https://github.com/crusaderky>`_.
 
+  - Fix bug when using ``pytest`` class decorators to skiping certain unittests.
+    The previous behavior unintentionally causing additional tests to be skipped.
+    A temporary work-around has been applied in :issue:`1531`.
```

Comment:
This is temporary for xarray, but from a user (or distributor) perspective the bug is fixed. I would probably drop that part.